### DISCUSSION TITLE
Not a bug-  only a question about Roku Search

### DISCUSSION THREAD

**Discussion by axelctv** [rrirower/Channel-Feed-Manager#150] [Q&A]
does anyone who is using this, manage to have a search feed published by Roku?

**Comment by podworx** [comment]
This was a very difficult process for us. We eventually did so, but with a ton of back-and-forth between us and Roku.

One thing; they told me the search feed generated by Channel Feed Manager was not compatible.  Specifically, the language "en-us" should be "en".  Once I changed it, we were able to get certified.

**Reply by rrirower** [reply]
I'm not sure what version you are using, but, that was resolved in version 1.0.1.2 .  See #153
